### DISCUSSION TITLE
Share with Customer

### DISCUSSION THREAD

**Discussion by fireblade95402** [microsoft/PubSec-Info-Assistant#391] [Q&A] [answered]
Hi 

My one customer is doing a lot with OpenAI and using an accelerator to build out 2 of their use cases. 

I came across the project and demo 'ed it to them. They loved it and asked me to share the link to the repo. I see it's private.

Can you please let me know if, when and how I could share this with my customer?

Cheers,

Mark

**Comment by lon-tierney** [comment, marked as answer]
Please send email to isat-support@microsoft.com
